portality/bll/services/events.py

```diff
@@ -26,32 +26,32 @@
 
 
 class EventsService(object):
-    # disabled events - to enable move the event to EVENT_CONSUMENRS array
+    # disabled events - to enable move the event to EVENT_CONSUMERS array
     DISABLED_EVENTS = [
```

Comment:
Disabling events will require an update to the tests in https://github.com/DOAJ/doaj/blob/develop/doajtest/unit/application_processors/test_application_processor_emails.py

Reply:
fixed